src/fonts.css

```diff
@@ -0,0 +1,18 @@
+@font-face {
+  font-family: "Noto Sans Condensed";
+  src: url("https://osm-americana.github.io/web-fonts/noto/NotoSans-CondensedMedium-PropNums.woff2")
```

Comment:
I opened osm-americana/web-fonts#1 as tail work for automating the creation of these files.